GETTING_STARTED.md

```diff
@@ -0,0 +1,127 @@
+
+### Spec Conformance Test
+
+Conformance tests should provide a clear, end-user runnable test suite for
+implementors to use to determine and demonstrate that an implementing project
+or product meets the explicit definitions of the specification.
+ - [OCI Conformance](https://github.com/opencontainers/oci-conformance)
+
+The most advanced conformance implementation to date is for the new distribution specification. Additional work on image and runtime conformance is ongoing.
+
+### Libraries 
+
+While hosting library code is not a common goal of the OCI, there are a few
+specific cases where small, non-end user focused, and tightly scoped libraries
+have been accepted into the OCI. The common denominator for these libraries are
+that they help implementors properly use features of the specifications:
+ - [go-digest](https://github.com/opencontainers/go-digest)
+ - [selinux](https://github.com/opencontainers/selinux)
+
+Utilities and end-user UX-oriented code is most likely better targeted at other
+more broad communities like the [CNCF](https://cncf.io). While there are not
+explicit rules, a discussion with the TOB is warranted for projects looking to
+contribute a library to OCI.
+
+### Reference Implementations
+
+While theoretically a specification can have one or more reference
+implementations, the OCI runtime spec and the program, `runc`, have gone
+hand in hand simply due to the particulars around the founding of the OCI.
+
+It is not inconceivable that more reference implementations would be 
+contributed and supported within the OCI, but at this point, the only
+active and viable reference implementation within the OCI is the `runc`
+implementation of the runtime specification, based around the contributed
+**libcontainer** codebase contributed by Docker.
+ - [runc](https://github.com/opencontainers/runc)
+
+Runc is also unique in that it is an open source reference implementation,
+but also a core dependent ecosystem component underneath the majority of
+container engine implementations in use today.
+
+For any future reference implementation to be adopted by the OCI, it would
+need to be kept in sync with the specification it implements. For a change
+to be accepted to the spec, the equivalent implementation would need to be
+authored and accepted as well.
+
+## Should my Project be in the OCI?
+
+The OCI receives proposals suggesting additions to the current suite
+of project types listed above. We understand that a perfect framework
+for determining inclusion or rejection of these proposals is an
+intangible goal. However, we list the following considerations to help
+guide future potential submissions.
+
+ 1. The OCI, unlike the CNCF, is not directly chartered for the advancement, marketing, and support of general cloud native software projects.
+ 2. Projects consumed via a UI and/or with a significant end user experience component are unlikely to be a good fit for the OCI model.
```

Comment:
1 really helps scope what you get with OCI. 
I'm struggling with 2. I think I see what you're saying isn't a good solution. Could we focus on what would? Supportive libraries or tools that enable the OCI specs, that need some level of endorsement to gain acceptance.

Reply:
This section is updated significantly; would love your re-review/comments.